src/encoded/static/components/filegallery.js

```diff
@@ -122,29 +125,30 @@ export class FileTable extends React.Component {
             context,
             items,
             graphedFiles,
+            filterOptions,
             filePanelHeader,
             encodevers,
-            selectedFilterValue,
-            filterOptions,
             showFileCount,
+            browserOptions,
             session,
             adminUser,
+            selectedFilterValue,
```

Comment:
It was a little confusing how these were rearranged, but I see that just the one parameter was added.